Docs: Plugin.json: Fix property descriptions, add missing properties, add example

robatron opened the pull request:
<!--

Thank you for sending a pull request! Here are some tips:

1. If this is your first time, please read our contribution guide at https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md

2. Ensure you include and run the appropriate tests as part of your Pull Request.

3. In a new feature or configuration option, an update to the documentation is necessary. Everything related to the documentation is under the docs folder in the root of the repository.

4. If the Pull Request is a work in progress, make use of GitHub's "Draft PR" feature and mark it as such.

5. If you can not merge your Pull Request due to a merge conflict, Rebase it. This gets it in sync with the master branch.

6. Name your PR as "<FeatureArea>: Describe your change", e.g. Alerting: Prevent race condition. If it's a fix or feature relevant for the changelog describe the user impact in the title. The PR title is used to auto-generate the changelog for issues marked with the "add to changelog" label.

-->

**What this PR does / why we need it**:

Fixes incorrect property descriptions, adds missing properties, and adds an up-to-date example.

robatron committed:
Fix property descriptions, add missing properties

Fixes incorrect property descriptions, adds missing properties, and adds an up-to-date example.

robatron commented:
Tip: Just learned about this "Rich Diff" for markdown files. Makes reviewing table changes much easier to read: https://github.com/grafana/grafana/pull/22281/files?short_path=6a0a273#diff-6a0a27323b123efafa45a3f60810de4d

oddlittlebird commented:
Thanks for the tip! That is great.

On Tue, Feb 18, 2020 at 1:07 PM Rob McGuire <notifications@github.com>
wrote:

> Tip: Just learned about this "Rich Diff" for markdown files. Makes
> reviewing table changes much easier to read:
> https://github.com/grafana/grafana/pull/22281/files?short_path=6a0a273#diff-6a0a27323b123efafa45a3f60810de4d
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/grafana/grafana/pull/22281?email_source=notifications&email_token=AMNF6KN6M6GYHHZCXF3CPTLRDREZBA5CNFSM4KXMHKJ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEME6ACY#issuecomment-587849739>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMNF6KJQF4KOZI3QRLH3CP3RDREZBANCNFSM4KXMHKJQ>
> .
>


-- 
Diana Payton (She/Her)
Sr. Technical Writer at Grafana Labs

briangann commented on docs/sources/plugins/developing/plugin.json.md:
```diff
@@ -12,17 +12,28 @@ weight = 8
+| info.keywords               | Array of plugin keywords. Used for search on grafana.com                                                                      |
+| info.links                  | An array of link objects to be displayed on this plugin's project page in the form `{name: 'foo', url: 'http://example.com'}` |
+| info.logos.small            | Link to small project logo                                                                                                    |
+| info.logos.large            | Link to large project logos                                                                                                   |
```
logos should be logo in description area (same as line above).  This should also be an SVG file, and they can be the same.

robatron replied:
Done, and done

robatron committed:
Update logo descriptions

Updates the "large" and "small" logo descriptions to specify SVG image type requirement, and that they may be identical

robatron commented:
Anything else I need to do with this PR?

srclosson reviewed (approved):
I'm good when the other feedback is all done.

robatron commented:
I implemented [the changes](https://github.com/grafana/grafana/pull/22281#discussion_r381035205) requested by @briangann .  Is there any other feedback?

oddlittlebird commented:
When merging is blocked due to changes that Git isn't quite clever enough to realize are made, I usually re-request a review from the reviewer. I just did that with @briangann for you.

robatron commented:
Ah, OK, thanks

briangann reviewed (approved):
lgtm!

robatron commented:
Looks like an unrelated check failed. Not sure how to fix that: https://circleci.com/gh/grafana/grafana/267477?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

aknuds1 commented:
@robatron I restarted the build, since it was due to an out-of-memory thing.

robatron commented:
Another unrelated failure: https://circleci.com/gh/grafana/grafana/275734?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

aknuds1 committed:
Merge remote-tracking branch 'origin/master' into patch-3

aknuds1 commented:
@robatron That last failure is because this branch hasn't merged in grafana/master in a while, and was missing a fix. I've merged with grafana/master and pushed to your branch, so should work now.

aknuds1 commented on docs/sources/plugins/developing/plugin.json.md:
```diff
@@ -12,17 +12,28 @@ weight = 8
+| info.updated                | Date when this plugin was built. Use `%TODAY%` for Grafana to autopopulate this value.                                        |
+| info.version                | Project version of this commit. Use `%VERSION%` for Grafana to autopopulate this value.                                      |
+
+# Plugin.json Example
```
Shouldn't this be a level 2 heading? I.e.,
```suggestion
## Plugin.json Example
```

robatron replied:
Yes it should, thank you

robatron committed:
Update docs/sources/plugins/developing/plugin.json.md

Co-Authored-By: Arve Knudsen <arve.knudsen@gmail.com>

robatron commented:
@aknuds1 , ready to merge now?

aknuds1 commented:
Merged - thanks for helping improve Grafana!
